.dockerignore

```diff
@@ -4,13 +4,8 @@ fly.toml
 tmp
 !tmp/pids
 log
-public/assets
-public/packs
 .bundle
 
-db/*.sqlite3
-db/*.sqlite3-*
-
```

Comment:
this commit is to remove unnecessary settings, mostly is about static files, because this app is for backend only